joint_state_publisher/joint_state_publisher/joint_state_publisher.py

```diff
@@ -406,13 +458,13 @@ def main():
     # Strip off the ROS 2-specific command-line arguments
     stripped_args = rclpy.utilities.remove_ros_args(args=sys.argv)
     parser = argparse.ArgumentParser()
-    parser.add_argument('urdf_file', help='URDF file to use', nargs='?', default=None)
+    parser.add_argument('description_file', help='Robot description file to use', nargs='?', default=None)
 
     # Parse the remaining arguments, noting that the passed-in args must *not*
     # contain the name of the program.
     parsed_args = parser.parse_args(args=stripped_args[1:])
```

Comment:
We can't do this exactly like this, because we would break backwards compatibility with anyone currently using the `urdf_file` argument.

Instead, I think we should deprecate the `urdf_file` argument, but still honor it for now.  Something like this (untested):

```suggestion
    parser.add_argument('urdf_file', help='URDF file to use (deprecated, use "description_file" instead)', nargs='?', default=None)
    parser.add_argument('description_file', help='Robot description file to use', nargs='?', default=None)

    # Parse the remaining arguments, noting that the passed-in args must *not*
    # contain the name of the program.
    parsed_args = parser.parse_args(args=stripped_args[1:])
    
    if parsed_args.urdf_file and parsed_args.description_file:
        raise Exception('Only one of "urdf_file" or "description_file" can be specified')
    elif parsed_args.urdf_file:
        warnings.warn('The "urdf_file" argument is deprecated, switch to using "description_file"')
        parsed_args.description_file = parsed_args.urdf_file
```

Reply:
My bad, fixed it in the new PR, still running some tests